packages/basic/index.js

```diff
@@ -7,18 +7,35 @@ import pino from 'pino'
 import { packageJson, schema } from './lib/schema.js'
 import { importFile } from './lib/utils.js'
 
+function isImportFailedError (error, pkg) {
+  if (error.code !== 'ERR_MODULE_NOT_FOUND') {
+    return false
+  }
+
+  const match = error.message.match(/Cannot find package '(.+)' imported from (.+)/)
```

Comment:
can we use an error code instead?

Reply:
We already do, see line 11.
This match is to check that the file that has failed the import is the stackable package and not something inside the package itself.